presto-native-execution/presto_cpp/main/http/filters/AccessLogFilter.h

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <fmt/format.h>
+#include <glog/logging.h>
+#include <proxygen/httpserver/Filters.h>
+#include <proxygen/httpserver/RequestHandlerFactory.h>
+#include <time.h>
+
+namespace facebook::presto::http::filters {
+
+/**
+ * A filter that does access logging in nginx `combined` format
+ */
```

Comment:
Use triple slash for comments. Check other classes for examples.
Also does it always log? Or we have choices to tune the verbose level?

Reply:
Yes, currently it logs all request to the server with level=INFO.
If we want to skip certain URIs, then that can be build in. But the intention is to log all requests submitted to the server (maybe exception can be health check URIs).

Reply:
I think this is not going to scale. Can we add a flag defaulting to not using that filter? This will cause prestissimo server to be spam with these logs. Presto does millions of ping requessts to Prestissimo server.

Reply:
Do you think prestissimo will benefit if we keep the AccessLog and skip only certain URIs? Or we should completely disable the filter?